Main.java

```diff
@@ -12,7 +12,8 @@ public static void main(String[] args) throws IOException {
         String birthdate = in.next();
         Person person1 = new Person(name,surname, patronymic,birthdate);
         person1.printInfo();
-        int veryveryveryveryverybadvariablename = 1;
+        int veryveryveryveryverybadvariablename=1;
+        int veryveryveryveryverybadvariablename2=1;
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'=' is not followed by whitespace.